build-tools/heroku/generate-syntax-keywords

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
wow impressive in a bash script (py would have been fine too)

Not sure I will ever be able to decipher this though haha

Reply:
The regex took a while to figure out, so I'm glad to hear you find it good.
I'll maybe add some per line comments so it's easier to figure out exactly what that script is doing.